### DISCUSSION TITLE
what do the values listed in the output from pathfindR signify

### DISCUSSION THREAD

**Discussion by obearap** [egeulgen/pathfindR#68] [Q&A]
Hi @egeulgen 

I'm using pathfindR to analyse data from my PhD. Reproduced below are the results obtained for 1 pathway which had only upregulated genes.

ID | Term_Description | Fold_Enrichment | occurrence | support | lowest_p | highest_p | Up_regulated
hsa03050 | Proteasome | 10.44726107 | 1 | 0.158974 | 2.23E-55 | 2.23E-55 | PSMD3, PSMD12, PSMD11, PSMD6,   PSMD7, PSMD13, PSMD8, PSMD4, PSMD1, PSME1, PSME2, PSMA6, PSMA2, PSMA4, PSMA7,   PSMA5, PSMA1, PSMA3, PSMB6, PSMB7, PSMB3, PSMB2, PSMB5, PSMB1, PSMB8 |  

One of the questions I've been asked by my supervisors is what does the fold enrichment, support and highest and lowest p values refer to. Do they simply refer to enrichment over the background species, ie the likely hood of these genes/pathways being present in my data by chance or do they refer to enrichment of expression, ie is this pathway 10.44 times more active in my data?

If it's the second option how does that apply when I have both upregulated and downregulated genes in a given pathway

Thanks

Apologies if the answer is elsewhere and I've missed it but any clarification that can be provided would be appreciated.

**Comment by egeulgen** [comment]
Hey @obearap,

Regarding what the output columns refer to:

Fold Enrichment: Fold enrichment value for the enriched term (Calculated using ONLY the input genes). This is calculated as the proportion of pathway genes within all INPUT (not subnetwork) genes over the proportion pathway genes within the background genes. For example, if the proportion of pathway genes in the input is 0.5 and the proportion of pathway genes in the background is 0.25, the fold enrichment value is 2. The background genes in pathfindR is used as all the genes in the protein-protein interaction network.

occurence: the number of iterations that the given term was found to enriched over all iterations (as the active subnetwork search + enrichment processes are run for multiple (by default 10) iterations)

support: the median support (proportion of active subnetworks leading to enrichment within an iteration) over all iterations. This reflects how many subnetworks support the enrichment

lowest and highest p values: these are the lowest and highest adjusted-p values of enrichment observed over all iterations

Hope this helps,
Best,
-E
